pkg/logging/logging_test.go

```diff
@@ -46,3 +54,69 @@ func TestSetGlobalLogLevel(t *testing.T) {
 	assert.Equal(t, GetGlobalLogLevel(), mDebug.GetLogLevel())
 	assert.Equal(t, GetGlobalLogLevel(), mInfo.GetLogLevel())
 }
+
+func Test_withRotatingCore(t *testing.T) {
+	t.Setenv(env.LoggingMaxBackups.EnvVar(), "2")
```

Comment:
Should we maybe add a test to show that files have been deleted? I know that we are testing the used library in that case, but just to be on the safe side. But after all, the reason for adding rotation is to avoid the growth of logs collected in a file (on storage).